File: cours8/exo1.js

function getDiscountPrice(price, discountPourcentage) {
  let newDiscountPrice = price - price * discountPourcentage / 100
  console.log("Prix final:" + newDiscountPrice)
  // Afficher newDiscountPrice
  return newDiscountPrice
}

// Appel de la fonction getDiscountPrice(12, 50)
let firstCustmers = getDiscountPrice(12, 50)

// J'aimerais transmettre le prix final au caissier, pour qu'il puisse demander la tune...
alert("Caisseir: OK TU ME DOIS " + firstCustmers + " Euros!")

//Appel getDiscountPrice(20, 30)
getDiscountPrice(20, 30)

File: cours8/exo1.py


def getDiscountPrice(price, discountPourcentage):
    newDiscountPrice = price - price * discountPourcentage / 100
    print("Prix total : " + str(newDiscountPrice))
    return newDiscountPrice

# Appel de la fonction getDiscountPrice(12, 50)
firstCustomer = getDiscountPrice(12, 50)

print('Caissier : OK TU ME DOIS ' + str(firstCustomer) + " Euros !")
